src/components/Chart/MainChart.js

```diff
@@ -158,7 +160,8 @@ class MainChart extends Component {
                                     onScenarioClickChart={this.handleScenarioClickChart}
                                 />
                                 <IndicatorSelection
-                                    statList={this.state.statList}
+                                    statList={this.state.statList} // TODO: can i remove this
+                                    STATS={this.props.STATS}
```

Comment:
If `STATS` is a constant, it shouldn't be passed down as a prop. If this prop is actually a parent's state, this variable should probably be renamed.

Reply:
Looks like it's being passed down as state and is not longer a const. Nevermind then.